openapi/bootstrap-agent.json

```diff
@@ -161,6 +161,45 @@
   },
   "components": {
     "schemas": {
+      "AllowedSourceIps": {
+        "description": "Description of source IPs allowed to reach rack services.",
+        "oneOf": [
+          {
+            "description": "Allow traffic from any external IP address.",
+            "type": "object",
+            "properties": {
+              "allow": {
+                "type": "string",
+                "enum": [
+                  "any"
+                ]
+              }
+            },
+            "required": [
+              "allow"
+            ]
+          },
+          {
+            "description": "Restrict access to a specific set of source IP addresses or subnets.\n\nAll others are prevented from reaching rack services.",
+            "type": "object",
+            "properties": {
+              "allow": {
+                "type": "string",
+                "enum": [
+                  "list"
+                ]
+              },
+              "ips": {
+                "$ref": "#/components/schemas/IpAllowList"
+              }
```

Comment:
at one point this was just the inline array schema. this is fine too; just pointing out the change as I'm not sure it was intentional.

Reply:
Yeah, that wasn't intentional. It'll be changed back in the next commit, since this one did _NOT_ blend.